siac/bashcomplcmd.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import "github.com/spf13/cobra"
+
+var (
+	bashcomplCmd = &cobra.Command{
+		Use:   "bash-completion [path]",
+		Short: "Creates bash completion file.",
+		Long:  `Creates a bash completion file at the specified location.
+
+Note: Bash completions will only work with the prefix with which the script is created (e.g. './siac' or 'siac').
+
+Once created, the file has to be moved to the bash completion script folder - usually /etc/bash_completion.d/`,
```

Comment:
Unfortunately not, unless `/etc/bash_completion.d/` can be edited by the current user. If the user calls `sudo siac bash-completion ...` then all completions will only work with the prefix `sudo siac` and not just `siac`.